tests/test_all.py

```diff
@@ -376,6 +376,13 @@ def test_no_orphaned_packages(container_per_test: ContainerData) -> None:
 @pytest.mark.parametrize(
     "container", CONTAINERS_WITH_ZYPPER_AS_ROOT, indirect=True
 )
+@pytest.mark.xfail(
```

Comment:
This seems to break tests on 15.4.

Reply:
yeah, because they're run on a non-suse host so the container-suseconnect integration is not working. rather than adding a quirk in the tests for this, maybe we should simply not define a repo like in the 16.0 case?